test/collection_test.js

```diff
@@ -174,6 +174,55 @@ describe("Collection", () => {
     });
   });
 
+  describe("#_setPlugins", () => {
+    var articles;
+
+    beforeEach(() => articles = testCollection());
+
+    it("should check that first argument is a supported type", () => {
+      try {
+        articles._setPlugins("bogus", {});
+        expect(false, "Should not reach here.");
+      } catch (err) {
+        expect(articles._plugins.remoteTransformers).to.be.instanceOf(Array);
+        expect(articles._plugins.remoteTransformers.length).to.equal(0);
+        expect(err.message).to.equal("Plugin type bogus not supported");
+      }
+    });
+
+    it("should set this._plugins.remoteTransformers", () => {
+      articles._setPlugins("remoteTransformers",
+          [ new QuestionMarkTransformer() ]);
+      expect(articles._plugins.remoteTransformers).to.be.instanceOf(Array);
+      expect(articles._plugins.remoteTransformers.length).to.equal(1);
+      expect(articles._plugins.remoteTransformers[0]).to.be.instanceOf(QuestionMarkTransformer);
```

Comment:
Nit: Generally speaking, we're trying to have a single assertion per test — with a proper label — so we can immediately identify which part of the expected functionality is broken.